app/api/v1/views.py

```diff
@@ -0,0 +1,128 @@
+"""This is the view for sendit application"""
+from flask import Flask, jsonify, make_response
+from flask_restplus import Api, Resource
+from .models import UserModel
+from .models import ParcelModel
+from .models import CheckRequired
+
+app = Flask(__name__)
+api = Api(app)
+
+
+class Users(Resource):
+    """This is the user class for signup"""
+    def post(self):
+        """The post request for the class"""
+        payload = api.payload
+        # check if payload is empty
+        check = CheckRequired(payload)
+        checked_payload = check.check_payload_signup()
+        if not checked_payload:
+            return {'status': 'failed', 'message': 'bad request refer to API document or provide all fields'}, 400
+        verifier = CheckRequired(payload)
+        email_checked = verifier.check_for_email()
+        if not email_checked:
+            return {'status': 'failed', 'message': 'bad request check your email'}, 400
+        email = payload['email']
+        user = UserModel()
+        data = user.get_user(email)
+        if data:
+            return {'status': 'failed', 'message': 'conflict email already used'}, 409
+        user = UserModel()
+        data = user.add_user(payload)
+        return {'status': 'added', 'message': 'Successfully signed up', 'data': data}, 201
+
+
+class Login(Resource):
+    """The login class that will handle login"""
+    def post(self):
+        """The post request for login"""
+        payload = api.payload
+        # Get the variables you'll use
+        if not payload:
+            return {'status': 'failed', 'message': 'provide all fields'}, 400
+        if not all(key in payload for key in ['email', 'password', 'role']):
+            return {'status': 'failed', 'message': 'bad request refer to API document'}, 400
+        email = payload['email']
+        role_provided = payload['role']
+        password_given = payload['password']
+        user = UserModel()
+        data = user.get_user(email)
+        if not data:  # returned from our model get_user
+            return {'status': 'failed', 'message': 'email not found'}, 401
+        role = data['role']
+        password = data['password']
+        if not password == password_given:
+            return {'status': 'failed', 'message': 'wrong password'}, 401
+        if role_provided == 'admin' and role == 'user':
+            return {'status': 'failed', 'message': 'you are not an admin'}, 403
+        payload = {"name": data['name'],
+                   "role": role,
+                   "user_id": data['id']
+                   }
+        return {'status': 'success', 'message': 'Successful logged in', 'date': payload}, 200
```

Comment:
Check the return statement it's `{ 'data': payload }` not  `{ 'date': payload }`